sdk/clientcore/core/src/main/java/module-info.java

```diff
@@ -8,6 +8,7 @@
     requires transitive java.xml;
 
     requires java.net.http;
+    requires jdk.httpserver;
```

Comment:
This seems to only be needed in testing. This should be done using the Surefire arglines and Failsafe arglines mutations of `javac` when running tests.

Reply:
this was not intentional, thanks!